### PR TITLE
Update election_vote_received_notification.rb

### PR DESCRIPTION
**Pull request by luzalbaposse**
feat: Enhance ElectionVoteReceivedNotification

- Added functionality to display the total number of received votes.
- Included candidate's position in the notification message.

## Summary

This commit enhances the `ElectionVoteReceivedNotification` class to provide additional information to users. It now displays the total number of votes received and informs the candidate of their current position in the voting.

## How to test?

U could test it like this:
```
RSpec.describe ElectionVoteReceivedNotification do
  let(:voter) { create(:user, name: "Vita") }
  let(:candidate) { create(:user, name: "Lu") } 
  let(:election) { create(:election, organization: create(:organization, slug: "ethereumfoundation")) }
  let(:vote) { create(:vote, voter: voter, election: election, candidate: candidate, amount: 5) }

  subject(:notification) { described_class.new(params: { "vote_id" => vote.id, "election_id" => election.id }) }

  describe "#body" do
    it "includes total votes received and candidate's position" do
      expect(notification.body).to include("You have received 5 votes from Lu.") 
      expect(notification.body).to include("Total votes received: 5.")
      expect(notification.body).to include("Your position is: #{notification.calculate_candidate_position(vote.candidate_id, election.id)}°")
    end
  end
end
```

## Notes
Not that sure of the test cause i'm new at ruby, but what i try to do is to create 2 users and provide votes between them

 glad to help:)

### CODE REVIEW

File: app/notifications/election_vote_received_notification.rb [hunk 1]
```diff
@@ -6,9 +6,19 @@ class ElectionVoteReceivedNotification < BaseNotification
...
+    "You have received #{source.amount} votes from #{source.voter.name}. Total votes received: #{total_votes_received}. Your position is: #{candidate_position}°"
   end
-
+  
+  def total_votes_received
+    Vote.where(election_id: params["election_id"]).sum(:amount)
+  end
+  
+  def calculate_candidate_position(candidate_id, election_id)
+    votes_received = Vote.where(election_id: election_id).order(amount: :desc).pluck(:candidate_id)
```
Comment (RubenSousaDinis) [inline app/notifications/election_vote_received_notification.rb:17]: Here you will need to group by the `candidate_id` and sum the `amount` of the votes. We create a new `Vote` record for each vote in the app.

**Comment by RubenSousaDinis** [comment]
We will display a leaderboard in the last two days of the voting. We're not doing that right now to be fair and give the same chances to all candidates.

Nevertheless, I still think your PR is valid and will provide useful information for the candidates!

File: app/notifications/election_vote_received_notification.rb (hunk 1 above)
Comment (RubenSousaDinis) [reply app/notifications/election_vote_received_notification.rb:17]: Also, feel free to attach the test file to the pull request

Comment (luzalbaposse) [reply app/notifications/election_vote_received_notification.rb:17]: Cool ! useful feedback

**Pull request closed without merging**